include/blisp.h

```diff
@@ -73,4 +73,6 @@ int32_t blisp_device_program_check(struct blisp_device* device);
 int32_t blisp_device_reset(struct blisp_device* device);
 void blisp_device_close(struct blisp_device* device);
 
-#endif
\ No newline at end of file
+#define UNUSED(x) (void)(x)
```

Comment:
Sidenote:
Root CMakeLists.txt requests C standard 23 via `CMAKE_C_STANDARD`. C23 introduced official `maybe_unused` attribute. Using latter might avoid clumsiness of ancient C helper construct and ease portability ("portable-across-compilers").
Ultimately, also matter of style.

Reply:
Thanks. I had no idea this made it to C standard, assumed it was C++ only. Didn't notice this project asked for C23, a pleasant surprise.

Reply:
Well, MSVC's C23 support is...what's the word I'm looking for...sparse. `[[maybe_unused]]` is not implemented, yet.
![image](https://user-images.githubusercontent.com/11135/235373879-6015224b-7fbd-4715-b222-f4b199d0c7e4.png)
From: https://en.cppreference.com/w/c/compiler_support/23